### PR TITLE
Bug 1037962 - Remove hard coded strings r=eragonj

### PR DESCRIPTION
**Pull request by crh0716**
https://bugzilla.mozilla.org/show_bug.cgi?id=1037962

### CODE REVIEW

**Comment by try-server-hook** [comment]
[crh0716 Arthur Chen (crh0716)](https://github.com/crh0716) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=0701113c6d14)

**Comment by EragonJ** [comment]
please check `settings/js/panels/about_more_info/device_info.js` too.

File: apps/settings/js/panels/simcard_manager/simcard_manager.js [hunk 1]
```diff
@@ -231,9 +232,20 @@ define(function(require) {
 
         // will generate ".sim-card-0 .sim-card-name" for example
         var targetSelector = cardSelector + ' .sim-card-' + selector;
-
-        this._elements.simCardContainer.querySelector(targetSelector)
-          .textContent = simcardInfo[selector];
+        var element =
+          this._elements.simCardContainer.querySelector(targetSelector);
+        if (selector === 'number') {
+          element.removeAttribute('data-l10n-id');
```
Comment (EragonJ) [inline apps/settings/js/panels/simcard_manager/simcard_manager.js]: This line may be redundant, right ?

Comment (crh0716) [inline apps/settings/js/panels/simcard_manager/simcard_manager.js]: Good catch!

**Comment by crh0716** [comment]
Turn to use innerHTML and bdi.

**Comment by try-server-hook** [comment]
[crh0716 Arthur Chen (crh0716)](https://github.com/crh0716) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=d7925ddaa7a6)

**Comment by try-server-hook** [comment]
[crh0716 Arthur Chen (crh0716)](https://github.com/crh0716) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=2ce5a3ea1dcb)

**Pull request merged**